src/parser/monk/mistweaver/modules/spells/SpinningCraneKick.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+
+import Analyzer from 'parser/core/Analyzer';
+import { formatMilliseconds } from 'common/format';
+
+const debug = false;
+
+class SpinningCraneKick extends Analyzer{
+    
+    goodSCKcount = 0;
+    goodSCKTimeList = [];
+    badSCKcount = 0;
+    badSCKTimeList = [];
+    canceledSCKcount = 0;//figure out if this is possible
+    enemiesHitSCK;
+    currnetTime = 0;
+
+    on_byPlayer_cast(event){
+        const spellId = event.ability.guid;
+
+        if(spellId === SPELLS.SPINNING_CRANE_KICK.id){
+            if(this.enemiesHitSCK){//this nested is needed due to weird logs
+                this.checkSCK();
+            }
+            this.currnetTime = this.owner.currentTimestamp - this.owner.fight.start_time;
+            this.enemiesHitSCK = [];
+        }
+    }
+
+    //tracking channel time isn't needed due to the fact it is the same as a gcd so they have to have another cast event
+    on_byPlayer_damage(event){
+        const spellId = event.ability.guid;
+
+        if(spellId === SPELLS.SPINNING_CRANE_KICK_DAMAGE.id && !this.enemiesHitSCK.includes(event.targetID)){
```

Comment:
the `if` checks for just `targetID` but the push adds `event.targetID + " " + event.targetInstance`.

Reply:
Just looked over it at the time but thanks you